playbook/app/pb_kits/playbook/pb_dialog/_dialog.jsx

```diff
@@ -37,7 +37,7 @@ type DialogProps = {
   opened: boolean,
   portalClassName?: string,
   shouldCloseOnOverlayClick: boolean,
-  size?: "sm" | "md" | "lg" | "content",
+  size?: "sm" | "md" | "lg" | 'status_size' | "content",
```

Comment:
```suggestion
  size?: "sm" | "md" | "lg" | "status_size" | "content",
```

This and a few other places using single ticks vs doubles.